### PR TITLE
Feature/arm skeleton

### PR DESCRIPTION
**Pull request by sw990**
created a lower and upper arm subsystem, as well a command to control arms with joysticks

### CODE REVIEW

File: src/main/java/competition/electrical_contract/ElectricalContract.java [hunk 1]
```diff
@@ -15,5 +15,13 @@ public abstract class ElectricalContract {
 
     public abstract DeviceInfo getSteeringEncoder(SwerveInstance swerveInstance);
 
+    public abstract DeviceInfo getLowerArmLeftMotor();
+
+    public abstract DeviceInfo getLowerArmRightMotor();
+    public abstract DeviceInfo getUpperArmLeftMotor();
+    public abstract  DeviceInfo getUpperArmRightMotor();
+    public abstract  boolean isArmReady();
```
Comment (JohnGilb) [inline src/main/java/competition/electrical_contract/ElectricalContract.java:23]: Please split this into `isUpperArmReady` and `isLowerArmReady` and make the relevant changes in the subsystems.

File: src/main/java/competition/subsystems/arm/commands/ControlArmsWithJoyStickCommand.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    @Inject
+    public ControlArmsWithJoyStickCommand(OperatorInterface oi, LowerArmSubsystem lowerArmSubsystem, UpperArmSubsystem upperArmSubsystem ){
+        this.oi = oi;
+        this.lowerArmSubsystem = lowerArmSubsystem;
+        this.upperArmSubsystem = upperArmSubsystem;
+
+    }
+
+    public void initialize(){
+
```
Comment (JohnGilb) [inline src/main/java/competition/subsystems/arm/commands/ControlArmsWithJoyStickCommand.java:26]: add this code:

`log.info("Initializing!");`. 

This lets us follow what the robot did in the logs after a match.

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: src/main/java/competition/electrical_contract/ElectricalContract.java (hunk 1 above)
Comment (sw990) [reply src/main/java/competition/electrical_contract/ElectricalContract.java]: Changes were made

File: src/main/java/competition/subsystems/arm/commands/ControlArmsWithJoyStickCommand.java (hunk 2 above)
Comment (sw990) [reply src/main/java/competition/subsystems/arm/commands/ControlArmsWithJoyStickCommand.java]: Code was added

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**